ports/onednn/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "onednn",
-  "version-semver": "2.6.1",
+  "version-semver": "3.0.0",
```

```suggestion
  "version": "3.0.0",
```
Is this port a semver library? If it is not, here should use `"version"`?